test/inductor/test_cpu_select_algorithm.py

```diff
@@ -333,6 +334,51 @@ def forward(self, x):
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
         self.assertEqual(counters["inductor"]["cpp_epilogue_fusion_counter"], 1)
 
+    @inductor_config.patch({"freezing": True})
+    @patches
+    @torch.no_grad
+    @unittest.skipIf(not TEST_MKL, "Test requires MKL")
+    @parametrize("batch_size", (32,))
+    @parametrize("in_features", (128,))
+    @parametrize("out_features", (64,))
+    @parametrize("bias", (False, True))
+    @parametrize("input_3d", (False, True))
+    @dtypes(torch.float32, torch.bfloat16)
+    def test_quantized_linear_with_pointwise(
+        self, batch_size, in_features, out_features, bias, input_3d, dtype
+    ):
+        B = (2, batch_size) if input_3d else (batch_size,)
+        input = torch.randn(*B, in_features).to(dtype=torch.float32)
+
+        class M(torch.nn.Module):
+            def __init__(self, bias):
+                super().__init__()
+                self.linear = torch.nn.Linear(in_features, out_features, bias)
+
+            def forward(self, x):
+                res = self.linear(x)
+                return res
+
+        counters.clear()
+        ref_quantized_mod = _generate_qdq_quantized_model(
+            M(bias=bias).eval(),
+            (input,),
+        )
+
+        atol, rtol = 1e-3, 1e-3
+        if dtype == torch.bfloat16:
+            atol, rtol = 1e-2, 1e-2
+
+        with patch.object(
+            select_algorithm, "VERIFY", dict(atol=atol, rtol=rtol)
+        ), torch.no_grad(), torch.autocast(
+            "cpu", enabled=(dtype == torch.bfloat16), dtype=dtype
+        ):
+            ref_res = ref_quantized_mod(input)
+            cfn = torch.compile(ref_quantized_mod)
+            res = cfn(input)
+            self.assertTrue(torch.allclose(ref_res, res, atol=atol, rtol=rtol))
```

Comment:
also checks `self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)`?

Reply:
Thanks and add the check.